packages/agent/conf/examples/templates/auckland/NotaryCompanyVerification.hbs

```diff
@@ -0,0 +1,12 @@
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1"
+    ],
+    "type": [
+        "VerifiableCredential",
+        "NotaryCompanyVerification"
+    ],
+    "description": "This credential proves that the company you represent is registered with the Van Auckland Notary",
```

Comment:
Description should be in the credential subject and made shorter. We need to be able to display it.

"Company has been verified by the notary"

Reply:
done